packages/fx-core/src/plugins/resource/appstudio/results.ts

```diff
@@ -8,9 +8,9 @@ export class AppStudioResultFactory {
   public static UserError(
     name: string,
     message: string,
-    innerError?: any,
+    helpLink?: string,
```

Comment:
why we changes the parameter sequence? it might cause problem if we didn't modify all function calls